draft-ietf-masque-h3-datagram.md

```diff
@@ -577,12 +297,22 @@ size advertised on that connection is too low), the intermediary SHOULD drop
 the HTTP Datagram instead of converting it to a DATAGRAM capsule. This
 preserves the end-to-end unreliability characteristic that methods such as
 Datagram Packetization Layer Path MTU Discovery (DPLPMTUD) depend on
-{{?RFC8899}}. An intermediary that converts QUIC DATAGRAM frames to datagram
+{{?RFC8899}}. An intermediary that converts QUIC DATAGRAM frames to DATAGRAM
 capsules allows HTTP Datagrams to be arbitrarily large without suffering any
 loss; this can misrepresent the true path properties, defeating methods such a
 DPLPMTUD.
 
 
+## The RELIABLE_DATAGRAM Capsule
+
+This document defines the RELIABLE_DATAGRAM capsule type (see {{iana-types}} for
+the value of the capsule type). This capsule has the same format and semantics
+as the DATAGRAM capsules, however it is opaque to intermediaries: intermediaries
+MUST NOT convert between QUIC DATAGRAM frames and RELIABLE_DATAGRAM capsules or
+vice versa. Endpoints MUST treat receipt of a RELIABLE_DATAGRAM capsule as
```

Comment:
what about RELIABLE_DATAGRAM -> DATAGRAM capsule conversion?

Maybe simpler to just state that they are passed through and MUST NOT be converted to anything else.